clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/CopyLastGoogleServerGroupAtomicOperation.groovy

```diff
@@ -260,7 +260,8 @@ class CopyLastGoogleServerGroupAtomicOperation extends GoogleAtomicOperation<Dep
     println(",, description autohealing policy ${description.autoHealingPolicy}")
     println(",, ancestor autohealing policy ${ancestorAutoscalingPolicy}")
     println(",, ancestor autohealing policy description ${ancestorAutoscalingPolicyDescription}")
-    newDescription.autoHealingPolicy = description.autoHealingPolicy ?: ancestorAutoHealingPolicyDescription
+    newDescription.autoHealingPolicy = (description.autoHealingPolicy != null || description.overwriteAncestorAutoHealingPolicy) ?
```

Comment:
Remove the `println`s. I think I snuck those in there.

Reply:
:+1: